mscore/palettebox.h

```diff
@@ -60,6 +61,13 @@ class PaletteBox : public QDockWidget {
       void clear();
       QList<Palette*> palettes() const;
       void updateWorkspaces();
+      QLineEdit* searchBox;
```

Comment:
Keep this one private please. If you need to access it from outside, rename it to `_searchBox` and create an accessor `searchBox()`